content/docs/1.3.0/advanced-resources/deploy/customizing-default-settings.md

```diff
@@ -42,41 +47,45 @@ From the project view in Rancher, go to **Apps > Launch > Longhorn** and edit th
       namespace: longhorn-system
     data:
       default-setting.yaml: |-
-        backup-target:
-        backup-target-credential-secret:
-        allow-recurring-job-while-volume-detached:
-        create-default-disk-labeled-nodes:
-        default-data-path:
-        replica-soft-anti-affinity:
-        storage-over-provisioning-percentage:
-        storage-minimal-available-percentage:
-        upgrade-checker:
-        default-replica-count:
-        default-data-locality:
-        guaranteed-engine-cpu:
-        default-longhorn-static-storage-class:
-        backupstore-poll-interval:
-        taint-toleration:
-        system-managed-components-node-selector:
-        priority-class:
-        auto-salvage:
-        auto-delete-pod-when-volume-detached-unexpectedly:
-        disable-scheduling-on-cordoned-node:
-        replica-zone-soft-anti-affinity:
-        node-down-pod-deletion-policy:
-        allow-node-drain-with-last-healthy-replica:
-        mkfs-ext4-parameters:
-        disable-replica-rebuild:
-        replica-replenishment-wait-interval:
-        disable-revision-counter:
-        system-managed-pods-image-pull-policy:
-        allow-volume-creation-with-degraded-availability:
-        auto-cleanup-system-generated-snapshot:
-        concurrent-automatic-engine-upgrade-per-node-limit:
-        backing-image-cleanup-wait-interval:
-        guaranteed-engine-manager-cpu:
-        guaranteed-replica-manager-cpu:
-        orphan-auto-deletion:
+        backup-target: s3://backupbucket@us-east-1/backupstore
+        backup-target-credential-secret: backup-target-credential-secret:
```

Comment:
These two settings should be in different lines

Reply:
Updated. Thank you.